.orchestra/config/components/xz.yml

```diff
@@ -0,0 +1,27 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@ source_url = "https://tukaani.org/xz/xz-5.4.0.tar.bz2"
+
+#@yaml/text-templated-strings
+---
+#@ def xz_args():
+license: COPYING
+configure: |
+  mkdir -p "$BUILD_DIR"
+  extract.sh --into "$BUILD_DIR" (@= source_url @)
+
+  cd "$BUILD_DIR"
+  ./configure --enable-werror --prefix="$ORCHESTRA_ROOT"
```

Comment:
Multiline, also usually for dependencies we don't check we don't enable `-Werror` it's just going to fire back at us if we bump the compiler and a new warning pops up.

Reply:
Done